cpp/tests/sampling/rw_low_level_test.cu

```diff
@@ -782,3 +782,121 @@ TEST_F(RandomWalksPrimsTest, SimpleGraphRandomWalk)
 
   ASSERT_TRUE(test_all_paths);
 }
+
+TEST_F(RandomWalksPrimsTest, SingleRandomWalk)
```

Comment:
Same feedback as last review: a proper gtest fixture should be used to drastically reduce the size of these tests and make them easier to maintain (especially since you're using `TEST_F`).

Reply:
Same answer as before: please describe the course of action desired, here. `TEST_F` doesn't compile without `RandomWalksPrimsTest` "fixture" (if that's what you're referring to).

Reply:
The request was to refactor the tests so they take advantage of a test fixture for reusing the repeated setup and teardown code.  I'm not proposing getting rid of `TEST_F` or the fixture name it requires.  [Here's](https://github.com/rapidsai/rmm/blob/branch-0.20/tests/device_buffer_tests.cu) a nice example of a test that uses a fixture (it's also using `TYPED_TEST` instead of `TEST_F`, but it's the same concept). It has just enough inlined in each test so you don't have to reverse engineer much to figure out what's going on, but the small amount of repeated setup which otherwise distracts from the test is handled by the fixture. Note also that almost every cuDF test uses a BaseFixture which handles the common RMM resource setup.

If you disagree that you need to refactor the repeated code into a fixture, then I'd recommend at least not using `TEST_F`, since it's misleading and unnecessary.  [Here's](https://github.com/rapidsai/rmm/blob/branch-0.20/tests/mr/device/cuda_async_mr_tests.cpp) an example of a test that is not using a fixture.

Reply:
Done.